tests/compilation_tests/grid_test.py

```diff
@@ -98,8 +101,20 @@ def test_node_boundaries(self):
         y_range = get_vincenty_y((lat_min, long_min), (lat_max, long_max))
         x_range = get_vincenty_x((lat_min, long_min), (lat_max, long_max))
         top_right_node = g.nodes[g.get_num_rows()-1][g.get_num_cols()-1]
-        self.assertLessEqual(top_right_node.get_m_coords()[0], x_range, "The meters grid shouldn't be larger than the lat bounds")
-        self.assertLessEqual(top_right_node.get_m_coords()[1], y_range, "The meters grid shouldn't be larger than the long bounds")
+        self.assertLessEqual(top_right_node.get_m_coords()[
+                             0], x_range, "The meters grid shouldn't be larger than the lat bounds")
+        self.assertLessEqual(top_right_node.get_m_coords()[
+                             1], y_range, "The meters grid shouldn't be larger than the long bounds")
+
+
+class TestInside(unittest.TestCase):
+    def test_inside(self):
+        lat_min, lat_max, long_min, long_max = 42.444250, 42.444599, -76.483682, -76.483276
+        g = Grid(lat_min, lat_max, long_min, long_max)
+        self.assertTrue(g.isInsideTriangle(2, 2, 3, 3, 1, 3, 2,
+                        2.5), "this point is inside the triangle")
+        self.assertFalse(g.isInsideTriangle(2, 2, 3, 3, 1, 3,
+                         10, 10), "this point is outside the triangle")
```

Comment:
I think it would be good to add this function inside of `TestGrid` since the extent of `TestInside` would be to test this helper function.

Reply:
Yep